internal/core/items.rs

```diff
@@ -1422,4 +1425,5 @@ i_slint_common::for_each_enums!(declare_enums);
 pub struct PointerEvent {
     pub button: PointerEventButton,
     pub kind: PointerEventKind,
+    pub repeated: i32,
```

Comment:
WPF for example has a count for press and release. On flutter you need to build it on your own e.g. using  `GestureDetector`. Qml has a `doubleClicked` callback and both `clicked` and `doubleClicked` are called.

Reply:
In QML the behavior appears to also depend on whether a doubleClicked handler is installed or not: https://tinyurl.com/2he6fdnu

Reply:
> > @ogoffart @tronical now you can check how this feel:

With "feel" I mean in a real world use-case, not just "debug" :-).

For our internal use, that means how would the code in `TextInput` look like to select the current word on double click and the paragraph on triple click.

For public use.... I'm wondering what kind of control or demo could serve as example? Can you think of any of our controls that we could try this out on?

> > ```
> > 
> >   TouchArea {  
> > 
> >       width: 100%;
> > 
> >       height: 100%;
> > 
> >       clicked => { debug("clicked"); }
> > 
> >       pointer-event(e) => {  
> > 
> >           debug(e.repeated);
> > 
> >       }
> > 
> >   }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > If this behavior is ok I will polish the PR otherwise I will do the refactoring to implement an other solution.
> 
> An alternative could be to use multiple only for click and use a callback parameter on the count.
> 
> ```
> TouchArea {
>     clicked(e) => {
>       debug(e.count);
>     }
> }
> ```

I think that `count` or `repeat-count` is certainly a better name that `repeated`.

Reply:
I think QML's behavior make sense.